src/components/NanoContract/NanoContractTransactionsList.component.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { useNavigation } from '@react-navigation/native';
+import React, { useState } from 'react';
+import {
+  StyleSheet,
+  View,
+} from 'react-native';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { NanoContractTransactionsListHeader } from './NanoContractTransactionsListHeader.component';
+import { NanoContractTransactionsListItem } from './NanoContractTransactionsListItem.component';
+import { COLORS } from '../../styles/themes';
+import { nanoContractAddressChangeRequest } from '../../actions';
+import { HathorFlatList } from '../HathorFlatList';
+
+const getNanoContractHistory = (ncId) => (state) => {
```

Comment:
Here you get the history from redux state but where are you filling this state? When do you request the full node for the history?

Reply:
I plan to do it on an `init` effect on `sagas/nanoContract` and I did it on PR 13.

Reply:
Added `init` here.